EMIP/1-1000/EMIP00816.xml

```diff
@@ -29,34 +29,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.9">
+<locus from="59r" to="61v" facs="059"/>
+<title xml:lang="gez" type="complete" ref="LIT6048OnPalmSundayRetuaH"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="62r" to="74v" facs="062"/>
+<title xml:lang="gez" type="complete" ref="LIT2149Onthe"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="75r" to="87r" facs="075"/>
+<title xml:lang="gez" type="complete" ref="LIT2118OntheD"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="87r" to="93r" facs="087"/>
+<title xml:lang="gez" type="complete" ref="LIT2109OntheA"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="93r" to="105v" facs="093"/>
+<title xml:lang="gez" type="complete" ref="LIT2137OntheP"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="105v" to="110r" facs="105"/>
+<title xml:lang="gez" type="complete" ref="LIT6047On12ApostlesRetuaH"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.15">
+<locus from="110r" to="115r" facs="110"/>
+<title xml:lang="gez" type="complete" ref="LIT2145OntheT"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.16">
+<locus from="115r" to="120v" facs="115"/>
+<title xml:lang="gez" type="complete" ref="LIT2124OntheF"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="121r" to="136v" facs="121"/>
+<title xml:lang="gez" type="incomplete" ref="LIT1750LahaMa"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+</msItem>
+
                </msContents>
+
                <physDesc>
                   <objectDesc form="Codex">
+
                      <supportDesc>
                         <support>
                            <watermark/>
                            <material key="parchment"/>
+                           <p>Parchment repair on <locus target="#45" facs="045"></locus>.</p>
                         </support>
                         <extent>
-                           <measure unit="leaf">i + 136</measure>
+                           <measure unit="leaf">136</measure>
+                           <measure unit="leaf" type="blank">1</measure>
                            <dimensions type="outer" unit="mm">
                               <height>260</height>
                               <width>230</width>
                               <depth>60</depth>
                            </dimensions>
                         </extent>
+                        <collation>
+                        <list>
+                        <item xml:id="q1" n="A"><dim unit="leaf">1</dim></item>
+                        <item xml:id="q2" n="1"><dim unit="leaf">8</dim><locus from="1r" to="8v"></locus></item>
+                        <item xml:id="q3" n="2"><dim unit="leaf">8</dim><locus from="9r" to="16v"></locus></item>
```

Comment:
you can include @facs statements on these @locus statements too, all will assist in navigating the image set